src/grimoirelab/core/scheduler/tasks/models.py

```diff
@@ -231,6 +231,8 @@ def prepare_job_parameters(self):
         task_args = {
             'storage_type': self.storage_type,
             'storage_url': self.task_args.get('storage_url'),
+            'storage_username': self.task_args.get('storage_username'),
+            'storage_password': self.task_args.get('storage_password'),
```

Comment:
I don't like to store credentials this way but for now, we can live with it. We should consider to have archivists as consumers instead of as tasks.